### PR TITLE
Update wasm_bindgen to 0.2.82

### PR DESCRIPTION
**Pull request by ImJeremyHe**
related to #1509

### CODE REVIEW

File: wasm_bindgen/3rdparty/BUILD.bazel [hunk 1]
```diff
@@ -29,7 +29,6 @@ crates_vendor(
                 "--cfg=ossl102h",
                 "--cfg=ossl110",
                 "--cfg=ossl110f",
-                "--cfg=ossl110g",
```
Comment (UebelAndre) [inline wasm_bindgen/3rdparty/BUILD.bazel:32]: Why was this dropped?

**Review by UebelAndre — CHANGES_REQUESTED** [review]
Thanks for putting this together! Just one question 😄

File: wasm_bindgen/3rdparty/BUILD.bazel (hunk 1 above)
Comment (ImJeremyHe) [reply wasm_bindgen/3rdparty/BUILD.bazel]: Sorry, It must be a misoperation. I updated this PR. Thank you!

**Review by UebelAndre — APPROVED** [review]
Thanks!

**Pull request merged**